StockTrader/src/main/java/com/restResource/StockTrader/entity/BuyTrigger.java

```diff
@@ -0,0 +1,36 @@
+package com.restResource.StockTrader.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.Min;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+
+/**
+ * Class for a BuyTrigger table entry.
+ */
+@Builder(toBuilder = true)
+@Entity
+@IdClass(CompositeKey.class)
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class BuyTrigger {
+    @Min(value = 0)
+    Integer stock_amount;
```

Comment:
can you change these to camel case to keep it consistent? (select and go to refactor->rename just in case you didn't know)

Reply:
Yep will do